target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-habanero-dvk.dts

```diff
@@ -0,0 +1,332 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/* Copyright (c) 2019, Robert Marko <robimarko@gmail.com> */
```

Comment:
I don't know the origin of the content, but previously there was LF copyright and it was licensed under MIT...